pkg/relayer/proxy/jsonrpc.go

```diff
@@ -78,4 +81,90 @@ func (j *jsonRPCServer) ServiceId() string {
 // when jsonRPCServer is used as an http.Handler with an http.Server.
 // (see https://pkg.go.dev/net/http#Handler)
 func (j *jsonRPCServer) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
```

Comment:
OPTIONAL NIT: Can we rename the receiver from `j` to `jsrv` to it's easy to read as `json service`